src/hardware/envelope.cpp

```diff
@@ -60,7 +60,7 @@ void Envelope::Update(const int frame_rate, const int peak_amplitude,
 	edge_increment = static_cast<float>(ceil_sdivide(peak_amplitude, expansion_phase_frames));
 
 //	DEBUG_LOG_MSG("ENVELOPE: %s grows by %-3f to %-5f across %-3u frames (%u ms)",
-//	              channel_name,
+//	              channel_name.c_str(),
```

Comment:
Feel free to turn this comment into and `#if 0` / `#endif` pair. I started using it and I actually like it better now.

Reply:
Fixed